node/resources/category.ts

```diff
@@ -0,0 +1,26 @@
+import { pick } from 'ramda'
+import { createXLSX, jsonToXLSXFields } from './xlsx'
+
+enum CategoriesTranslatableField {
+  metaTagDescription = 'MetaTagDescription',
+  name = 'name',
+  title = 'Title',
+}
+
+const getBrandTranslatableFields = pick(Object.values(CategoriesTranslatableField))
+
+type CategoriesTranslatables = Record<CategoriesTranslatableField, string>
+
+const jsonToXLSXField: CategoriesTranslatables = {
+  MetaTagDescription: '_MetaTagDescription',
+  Title: '_Title',
+  name: '_Name',
+}
+
+export async function getCategoryXLSX(ctx: Context) {
+  const { clients: { catalog } } = ctx
+  const categories = await catalog.exportCategories() as Array<Record<string, string>>
```

Comment:
⛏ In my opinion, it'd be better to type the `exportCategories` method instead of having to typecast its result whenever it's used.